mbus-api/src/v0.rs

```diff
@@ -182,3 +202,276 @@ pub struct Node {
 #[derive(Serialize, Deserialize, Default, Debug, Clone)]
 pub struct Nodes(pub Vec<Node>);
 bus_impl_message_all!(GetNodes, GetNodes, Nodes, Node);
+
+/// Pool Service
+/// Filter query objects
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum Filter {
+    /// All objects
+    All,
```

Comment:
Filter all means, all objects so in effect the variant should be None?

Reply:
Hmm good point :+1: